plugins/utils/hmts.nix

```diff
@@ -0,0 +1,26 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.hmts;
+  helpers = import ../helpers.nix {inherit lib;};
+in {
+  options.plugins.hmts =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "hmts.nvim";
+
+      package = helpers.mkPackageOption "hmts.nvim" pkgs.vimPlugins.hmts-nvim;
+    };
+
+  config = mkIf cfg.enable {
+    extraPlugins = [cfg.package];
+
+    extraConfigLua = ''
+      require('hmts').setup()
+    '';
```

Comment:
I think we don't need that:
https://github.com/calops/hmts.nvim#faq

Reply:
yeh i also thought about leaving it out! wasnt sure if it better to be more consistent with other plugins!

Reply:
should be fixed now.